src/orange/surf/detail/InvoluteSolver.hh

```diff
@@ -44,11 +44,18 @@ class InvoluteSolver
     //! \name Type aliases
     using Intersections = Array<real_type, 3>;
 
+    //! Enum defining chirality of involute
+    enum Sign
+    {
+        CLOCKWISE = 1 == 1,
+        ANTICLOCKWISE = 1 != 1
```

Comment:
```suggestion
    enum Sign : bool
    {
        clockwise = 0,
        anticlockwise,
```

I'm curious about the semantics of your definitions as originally written though...